course-mw/course-actors-common/src/main/java/org/sunbird/userorg/UserOrgServiceImpl.java

```diff
@@ -228,4 +233,20 @@ public void sendEmailNotification(Map<String, Object> request, String authToken)
     }
     return null;
   }
+
+  private CompletableFuture<Map<String, Object>> getUserDetail(String userId, String authToken) {
+    logger.info(null, "UserOrgServiceImpl::getUserDetail::called");
+    CompletableFuture<Map<String, Object>> future = CompletableFuture.supplyAsync(new Supplier<Map<String, Object>>() {
+      @Override
+      public Map<String, Object> get() {
+        Map<String, Object> userDetail = getUserById(userId, authToken);
+        return new HashMap<>() {{
+          put(JsonKey.ID, userDetail.get(JsonKey.ID));
+          put(JsonKey.FIRST_NAME, userDetail.get(JsonKey.FIRST_NAME));
+          put(JsonKey.LAST_NAME, userDetail.get(JsonKey.LAST_NAME));
+        }};
```

Comment:
return back the entire userDetails from here. Filter out required fields in calling method